File: Java/Maths/Swap two input numbers or variables.java
import java.util.Scanner;
class swapping_input {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number for x: ");
        int x = input.nextInt();
        System.out.print("Enter a number for y: ");
        int y = input.nextInt();

        int temp = x;
        x = y;
        y = temp;
        
        System.out.println("\nAfter swap: ");
        System.out.println("x is = " + x);
        System.out.println("y is = " + y);
            
    }
}

File: C++/Maths/Swap two input numbers or variables.cpp
#include <iostream>
using namespace std;

int main(){
    int x, y;
    cout << "Enter a number for x: ";
    cin >> x;
    cout << "Enter a number for y: ";
    cin >> y;
    
    int temp = x;
    x = y;
    y = temp;

    cout << "\nAfter swap: " << endl;
    cout << "x is = " << x << endl;
    cout << "y is = " << y;
}